### PR TITLE
release custom CommCare logo uploader

### PR DESCRIPTION
**Pull request by nickpell**
Releases: http://manage.dimagi.com/default.asp?111183

Passed QA: http://manage.dimagi.com/default.asp?153736

:tada: :fireworks: :beers: :dancers: :balloon:

### CODE REVIEW

**Comment by czue** [comment]
congrats!

File: corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml [hunk 1]
```diff
@@ -75,14 +75,12 @@
   description: "Upload a file to serve as a CommCare home screen logo on Android phones"
```
Comment (dannyroberts) [inline corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml]: In the tickets it said Java and Android, here it specifically just says Android. Is this one accurate?

Comment (nickpell) [inline corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml]: Yes, the decision was made to not support Java unless a client specifically requests it.

Comment (dannyroberts) [inline corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml]: :+1:

**Pull request merged**